### PR TITLE
README.md: Skip follow-up steps if there is no pull request

### PR DESCRIPTION
**Pull request by hartwork**
Just an idea for consideration :beers:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ Note that in order to read the step outputs the action step must have an id.
         id: cpr
         uses: peter-evans/create-pull-request@v3
       - name: Check outputs
+        if: ${{ steps.cpr.outputs.pull-request-url }}
```
Comment (peter-evans) [inline README.md:93]: I would prefer to use `pull-request-number` here for consistency.
```suggestion
        if: ${{ steps.cpr.outputs.pull-request-number }}
```

**Review by peter-evans — CHANGES_REQUESTED** [review]
I deliberately tried to keep the examples very simple and just the bare minimum. However, I'm happy to accept these additions because it's likely that most use cases will need to add them anyway.

Just one minor change and I'll merge this in. Thanks 👍

File: README.md (hunk 1 above)
Comment (hartwork) [reply README.md:93]: Okay, why not :smiley: :+1:

Comment (hartwork) [reply README.md]: Done, pushed.

**Comment by peter-evans** [comment]
Thanks!

**Pull request merged**